README.md

```diff
@@ -13,36 +13,25 @@ Currently available subcommands:
 [![Coverage Status](https://coveralls.io/repos/killercup/cargo-edit/badge.svg?branch=master&service=github)](https://coveralls.io/github/killercup/cargo-edit?branch=master)
 [![crates.io](https://img.shields.io/crates/v/cargo-edit.svg)](https://crates.io/crates/cargo-edit)
 
+## Contribution
 
-## Installation
-
-### Using `cargo install`
-
-If you have a recent version of `cargo`, you can use `cargo install` to get all the tools provided by `cargo-edit` in one simple step:
+Thanks for your interest - we gratefully welcome contributions.
 
-```sh
-$ cargo install cargo-edit
-```
+Questions can be asked in [issues](https://github.com/killercup/cargo-edit/issues), or on [Gitter](https://gitter.im/cargo-edit/Lobby).
```

Comment:
Yeah, I created it some time ago but it's not been used much. I get notifications fir it though (I think). Adding a badge would be cool.